Makefile

```diff
@@ -0,0 +1,36 @@
+BUILD_DIR = build
+SERVICES = edgex-app
+CGO_ENABLED ?= 0
+GOOS ?= linux
+
+define compile_service
+	CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH) GOARM=$(GOARM) go build -ldflags "-s -w" -o ${BUILD_DIR}/edgex-app cmd/main.go
+endef
+
+all: $(SERVICES)
+
+.PHONY: all $(SERVICES)
+
+
+$(SERVICES):
+	$(call compile_service,$(@))
+
+clean:
+	rm -rf ${BUILD_DIR}
+
+install:
+	cp ${BUILD_DIR}/* $(GOBIN)
+
+test:
+	GOCACHE=off go test -v -race -tags test $(shell go list ./... | grep -v 'vendor\|cmd')
+
+changelog:
```

Comment:
I do not believe that you need to take care about changelog at this point. You can drop this one for simplicity.